dnnsoftware/Dnn.Platform
# Add 'fileName' to condition when checking request for logo file.

**Test Steps**
1. On PersonaBar select Site Settings.
2. Click left icon in Site Logo area. 
3. Area changes to two drop downs Folder and File. 
4. Select "None Specified" in File drop down.
5. Press ENTER or click [ENTER]
6. Click the 'Save' button.
     Expected Result: "Settings have been updated" message should appear without error.
7. Logout
8. Log back in.
9. On PersonaBar select Site Settings.
10. View Site Logo area. 
     Expected Results:  No logo should be displayed in the Site Logo area.

Fixes [#3095](https://github.com/dnnsoftware/Dnn.Platform/issues/3095) 

## Summary
The issue here is that the 'folderPath' property on the 'LogoFile' request object is populated with the path that the user selects from the 'Folder' drop-down list, making the request.LogoFile object != null.  Because request.LogoFile was not null, the previous logic attempted to retrieve the file even when 'None Specified' was selected from the 'File' drop-down list.  I added 'fileName' to the condition to ensure that a file is selected from the 'File' drop-down list before attempting to retrieve the file.
